core/lib/resources/presence/index.js

```diff
@@ -232,6 +232,8 @@ Presence.prototype.broadcast = function(message, except) {
         'explicit:', (socket && socket.id && self.subscribers[socket.id]), self.name);
     }
   });
+
+  this.emit('message:outgoing', message);
```

Comment:
why presence only? Also, this will repeat right? (Each server interested in this resource will send one)